From: Hilko Bengen <bengen@debian.org>
Date: Fri, 30 May 2014 20:55:30 +0200
Subject: Use Debian's ca-certificates

---
 ncat/ncat_posix.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ncat/ncat_posix.c b/ncat/ncat_posix.c
index b9fc3bc..40c731f 100644
--- a/ncat/ncat_posix.c
+++ b/ncat/ncat_posix.c
@@ -347,7 +347,7 @@ void set_lf_mode(void)
 
 #ifdef HAVE_OPENSSL
 
-#define NCAT_CA_CERTS_PATH (NCAT_DATADIR "/" NCAT_CA_CERTS_FILE)
+#define NCAT_CA_CERTS_PATH "/etc/ssl/certs/ca-certificates.crt"
 
 int ssl_load_default_ca_certs(SSL_CTX *ctx)
 {
